reorder_python_imports.py

```diff
@@ -520,29 +520,39 @@ def _report_diff(contents, new_contents, filename):
     print(diff, end='')
 
 
-FUTURE_IMPORTS = (
-    ('py22', ('nested_scopes',)),
-    ('py23', ('generators',)),
-    ('py26', ('with_statement',)),
+VERSION_IMPORTS = (
+    ('py22', ('nested_scopes',), ''),
+    ('py23', ('generators',), ''),
+    ('py26', ('with_statement',), ''),
     (
         'py3',
         ('division', 'absolute_import', 'print_function', 'unicode_literals'),
+        '',
     ),
-    ('py37', ('generator_stop',)),
-)  # type: Tuple[Tuple[str, Tuple[str, ...]], ...]
+    (
+        'py33',
+        ('division', 'absolute_import', 'print_function', 'unicode_literals'),
```

Comment:
shouldn't need to duplicate this here, I assume py33 implies py3?

Reply:
Let me test this some more locally